.github/workflows/label.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will triage pull requests and apply a label based on the
```

Comment:
How is this file in your branch? This was supposed to be in master. Did you rebase master instead?

Reply:
i was curious about this as well

Reply:
I am using the github app and since the master is the default branch, this file must have been picked up from there.

Reply:
it should be removed. only add files you want to a PR. when making a feature branch it should be based off of dev not master

Reply:
I spoke to Rohit, he mentioned that for this a revert is not necessary